modules/admin_manual/pages/enterprise/security/oidc/index.adoc

```diff
@@ -0,0 +1,109 @@
+= OpenID Connect (OIDC)
+:toc: right
+:toclevel: 2
+:panva-node-oidc-provider-url: https://github.com/panva/node-oidc-provider
```

Comment:
We Shall Not talk about the nodejs openid connect provider. This thing only for dev purpose.